### ISSUE TITLE
Create a twig function for getting the result object

### ISSUE THREAD

**Issue by ThomasLandauer** [endroid/qr-code-bundle#44]
It would be nice to be able to fill the `width` and `height` attribute of the `<img>`.
Still remembering the discussion at https://github.com/endroid/qr-code/issues/95: Am I right that when setting `size: 137` in `config/packages/endroid_qr_code.yaml`, I can't be sure that the resulting QR code will have exactly 137 width and height?

If so, then I'd suggest to find a way to pass the actual size to the user. I even had an idea how this could look like: Create a function `qr_code_object()` that works like this:
```twig
{% set qr_code = qr_code_object(...) %}
<img src="{{ qr_code.dataUri }}" width="{{ qr_code.width }}" height="{{ qr_code.height }}">

**Comment by endroid** [comment]
I think this one can still be useful. I will look into this myself. Thank you! We will be working with the result here so I think the way to go here is adding a getMatrix() method to the resultinterface and passing the matrix to each result. At this point the matrix is an internal implementation detail but this change will enable developers to post-process results easier.

**Comment by endroid** [comment]
Implemented in 4.1.1.

**Comment by ThomasLandauer** [comment]
OK, so is this the recommended usage now?:
```twig
{% set qr_code = qr_code_result('foobar') %}
<img src="{{ qr_code.dataUri }}" width="{{ qr_code.matrix.innerSize }}" height="{{ qr_code.matrix.innerSize }}">
```
Or is there a simpler way?
